compiler/rustc_infer/src/infer/nll_relate/mod.rs

```diff
@@ -114,7 +114,7 @@ pub trait TypeRelatingDelegate<'tcx> {
 }
 
 #[derive(Copy, Clone)]
-struct UniversallyQuantified(bool);
+struct UniversallyQuantified(#[allow(unused_tuple_struct_fields)] bool);
```

Comment:
That one is getting nuked by https://github.com/rust-lang/rust/pull/118257, you may as well remove the whole struct.

Reply:
Sounds good; I'll wait until that is merged to avoid rework / conflicts.